bundler.d/assets.rb

```diff
@@ -1,21 +1,21 @@
 group :assets do
   gem 'ace-rails-ap', '~> 4.0.0'
-  gem 'sass-rails', '~> 3.2'
+  gem 'sass-rails'
   gem 'uglifier', '>= 1.0.3'
   gem 'execjs', '>= 1.4.0', '<2.5.0'
   gem 'jquery-rails', '2.0.3'
   gem 'jquery-ui-rails', '< 5.0.0'
   gem 'bootstrap-sass', '3.0.3.0'
-  gem 'spice-html5-rails', '~> 0.1.4'
+  gem 'spice-html5-rails', '~> 0.1.5'
   gem 'flot-rails', '0.0.3'
   gem 'quiet_assets', '~> 1.0'
-  gem 'gettext_i18n_rails_js', '~> 0.0', '>= 0.0.8'
+  gem 'gettext_i18n_rails_js', '~> 1.0.0'
   # unspecified dep of gettext_i18n_rails_js
   #   https://github.com/nubis/gettext_i18n_rails_js/pull/23
   gem 'gettext', '~> 3.1', :require => false
   gem 'multi-select-rails', '~> 0.9'
-  gem 'gridster-rails', '~> 0.1'
-  gem 'jquery_pwstrength_bootstrap', '~> 1.2'
+  gem 'gridster-rails', github: "hampei/gridster-rails", branch: "rails4" # TODO
+  gem 'jquery_pwstrength_bootstrap', github: "unorthodoxgeek/jquery_pwstrength_bootstrap-gem" #TODO
```

Comment:
@hampei thanks very much!

http://projects.theforeman.org/issues/12418 should be fixed on develop first either to provide compatibility with 0.2.x and higher or to lock to 0.1.5.1.